src/core/doh.js

```diff
@@ -24,7 +24,9 @@ async function proxyRequest(event) {
   try {
     const plugin = new RethinkPlugin(event);
     await plugin.initIoState(io);
-
+    if (io.httpResponse) {
+      return io.httpResponse;
```

Comment:
Need to optionally set cors headers... That's okay I guess. This entire business with `io-state` needs a refactor...

Reply:
Thanks, I've added optional cors headers now 👍